src/content/docs/dev/send-tokens/sui-its.mdx

```diff
@@ -0,0 +1,502 @@
+1. `ctx`: The [transaction context](https://docs.sui.io/references/framework/sui/tx_context#sui_tx_context_TxContext) provides the necessary runtime environment for creating or modifying objects and state
+
+```move
+public fun receive_interchain_transfer<T>(
+    self: &mut ITS,
+    approved_message: ApprovedMessage,
+    clock: &Clock,
+    ctx: &mut TxContext,
+) {
+    let value = self.value_mut!(b"receive_interchain_transfer");
+
+    value.receive_interchain_transfer<T>(approved_message, clock, ctx);
+}
+```
+
+Once the relayer triggers this function it in turn triggers the [give_coin()](give-coin) function on the Coin Management program. Then once `give_coin()` has run the function will transfer the coin to the destination address.
+
+The full `receive_interchain_transfer()`  can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L247) 
+
+
+#### Receive Interchain Transfer With Data
+
+If the source chain is sending executable metadata along with the transaction that data will be handled by the `receive_interchain_transfer_with_data()` function. 
+
+It takes the same parameters as the previous `receive_interchain_transfer()` function except it also includes a `channel` parameter. The channel is used to check if the destination address in the payload matches the channel's own address, ensuring that messages with extra data are properly routed.
+
+The key differences with `receive_interchain_transfer_with_data()`, are that the function `asserts` that the `data` being sent is not empty and it does not simply transfer the coin to the destination address. This function is designed for transfers that carry extra information and requires additional routing and validation steps, while the standard version is for simple transfers that don’t include extra data and performs the transfer immediately. It is up to the caller to decide how to transfer the coin once the function returns
+
+```move
+public fun receive_interchain_transfer_with_data<T>(
+    self: &mut InterchainTokenService,
+    approved_message: ApprovedMessage,
+    channel: &Channel,
+    clock: &Clock,
+    ctx: &mut TxContext,
+): (String, vector<u8>, vector<u8>, Coin<T>) {
+    let value = self.value_mut!(b"receive_interchain_transfer_with_data");
+
+    value.receive_interchain_transfer_with_data<T>(
+        approved_message,
+        channel,
+        clock,
+        ctx,
+    )
+}
+```
+
+See here for an [example](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/sui/its-example.js#L124) of how to receive a transfer with data.
+
+The full `receive_interchain_transfer_with_data()` implementation can be found [here](https://github.com/axelarnetwork/axelar-cgp-sui/blob/main/move/interchain_token_service/sources/versioned/interchain_token_service_v0.move#L282)
+
+## Coin Management
```

Comment:
Please mention that an operator and distributor can be added to this before coin registration.

Reply:
resolving to unblock (need to get this in asap). Followed up in message so we can get this fixed after